### DISCUSSION TITLE
Export to CSV

### DISCUSSION THREAD

**Discussion by defactomail** [YOURLS/YOURLS#3361] [General]
I'm not sure if I'm missing it, but it would be great to be able to export the database of links for backup and migration purposes. 

Is is presently possible to do so, and I'm just missing it?

Thanks

**Comment by bmercernccer** [comment]
Like many other functions, this is done via a plugin. 
Should be available from the plugins list, here... 
https://github.com/YOURLS/awesome-yourls

**Comment by defactomail** [comment]
Thanks. I will try that. I initially looked in plugins, but didn't see import/export. I didn't realize there were many more plugins other than the ones that came with the install.

Thanks again
